docs/guides/use-shared-project-for-organization-wide-collaboration.md

```diff
@@ -9,81 +9,83 @@ import BulkAddUserImg from './img/invite-all-organization-members.png';
 
 ## About this guide
 
 This guide shows how you can consolidate these perspectives into a single shared project, ensuring clarity, reusability, and governance within your organization’s efforts.
 
 Teams can use shared projects to:
 
-- Prevent duplicate work.
-- Maintain a single source of truth.
-- Enable cross-project navigation and transparency.
-- Govern access to organization resources effectively.
+- Prevent duplicate work
+- Maintain a single source of truth
+- Enable cross-project navigation and transparency
+- Govern access to organization resources effectively
 
-## Create a shared project
+## Create a shared project and invite collaborators
 
-To create a shared project as a repository for your organization's resources:
+If you have not yet already, [create a Camunda 8 account](/guides/create-account.md)
 
-Open Modeler, and select **Create new project** to create a new project to store resources in.
+To create a shared project as a space for your organization's resources:
 
-![web modeler empty home](../components/modeler/web-modeler/img/web-modeler-new-user-home.png)
+1. Open [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md) and select **Create new project** (if you do not have any existing projects) or **New project** (if you already have existing projects).
+2. Name your project accordingly.
+   ![web modeler empty home](../components/modeler/web-modeler/img/web-modeler-new-user-home.png)
+3. On the right side of the page under **Collaborators**, select **Add user**.
+4. Enter the email address of the collaborator you would like to invite and select `Enter` or `Tab` between each entry. You may enter up to 20 email addresses. You can also use the **All users in the organization** option to send a bulk invitation to all colleagues.
+   <img src={BulkAddUserImg} alt="Add all organization users" width="750px"/>
+5. (optional) Enter an invitation message in the text box below your invites.
+6. Select **Send invite**. This will invite the users to both the organization and project. Once your colleague clicks **Accept invitation**, they can access the shared project with the assigned role and permissions.
+
+## Assign project roles to collaborators
 
-## Invite collaborators to a shared project
+By default, invited collaborators will become **Editors**. To understand what permissions each role has, review [access rights and permissions](/components/modeler/web-modeler/collaboration.md#access-rights-and-permissions).
 
-If you want to invite collaborators to your shared project, you must also ensure you assign them the correct permissions and roles.
+1. To change the role of a user, under **Collaborators** select the three vertical dots to the right of the collaborator name.
+2. Select **Change role**. You may select Project Admin, Editor, Commenter, or Viewer.
 
-### Add user to organization
+## Assign organization roles to collaborators
```

Comment:
IMO this page should be focused on the assets in the project, not managing the people. Changing the organization roles can best be handled as a link